.travis.yml

```diff
@@ -4,5 +4,9 @@ jdk:
 before_script:
```

Comment:
Please remove the before_sctipt part, this project has no database related functionlity.